src/app/modules/form/admin-form/__tests__/admin-form.service.spec.ts

```diff
@@ -53,42 +55,23 @@ import {
 
 import { generateDefaultField } from 'tests/unit/backend/helpers/generate-form-data'
 
+import { SMS_WARNING_TIERS } from '../../../../../shared/util/verification'
+import { smsConfig } from '../../../../config/features/sms.config'
+import MailService from '../../../../services/mail/mail.service'
+import * as SmsService from '../../../../services/sms/sms.service'
 import {
   FormNotFoundError,
   LogicNotFoundError,
   TransferOwnershipError,
 } from '../../form.errors'
+import * as FormUtils from '../../form.utils'
 import {
   CreatePresignedUrlError,
   EditFieldError,
   FieldNotFoundError,
   InvalidFileTypeError,
 } from '../admin-form.errors'
-import {
-  archiveForm,
-  createForm,
-  createFormField,
-  createFormLogic,
-  createPresignedPostUrlForImages,
-  createPresignedPostUrlForLogos,
-  deleteFormField,
-  deleteFormLogic,
-  disableSmsVerificationsForUser,
-  duplicateForm,
-  duplicateFormField,
-  editFormFields,
-  getDashboardForms,
-  getFormField,
-  reorderFormField,
-  transferFormOwnership,
-  updateEndPage,
-  updateForm,
-  updateFormCollaborators,
-  updateFormField,
-  updateFormLogic,
-  updateFormSettings,
-  updateStartPage,
-} from '../admin-form.service'
+import * as AdminFormService from '../admin-form.service'
```

Comment:
actual tests are at the bottom of this file; this change was made to allow the usage of `jest.spyOn`